settings.d/ansible.yml.example

```diff
@@ -1,4 +1,9 @@
 ---
 :enabled: true
 :working_dir: '~/.foreman-ansible'
+:vcs_integration: true
```

Comment:
When setting `:vcs_integration: false`, I can still trigger the role cloning from the GUI. However, the task remains stuck in the `pending` status.
I believe the intended behavior here is to restrict access to the new form, correct?

Reply:
Yes, that would be ideal but iirc, the functionality to inform Foreman of enabled/disabled _features_ is not yet a thing as described in the last paragraph of @ekohl's comment here:
https://github.com/theforeman/smart_proxy_ansible/pull/85#pullrequestreview-1799663079